### ISSUE TITLE
Label text is not appearing in the latest version 1.0.0

### ISSUE THREAD

**Issue by roginator** [ChuckJHardy/ColorTitlesTrello#6]
After agreeing to the increased permissions, the label text is not appearing on the cards.

**Comment by gitawego** [comment]
same here

**Comment by signal-intrusion** [comment]
yup broken. sorry.

**Comment by xandermathews** [comment]
**TL;DR**
Try going to your extensions page, opening the just-created "options" panel for this plugin, checking a box or more as desired, and hit save.

Then reload the trello page.

(tested on chrome only.)

**discovery**
I'm a user who hit the same blocker.

I'm reasonably sure I've figured it out while poking around in the [sudden extra permissions issue](https://github.com/ChuckJHardy/ColorTitlesTrello/issues/5)

The root cause is that when 1.0.0 changed all of the plugin's features to toggleable, it introduced 3 checkboxes, but the configuration storage naturally has none of these brand-new plugin-specific flags set.

So for anyone who upgrades to 1.0.0, the effective default for every feature has switched from on to off.

**brainstorm for better upgrade UX**
@ChuckJHardy 
if options.js _always_ sets a fourth option "user_has_used_option_panel" to true,
background.js can check for this flag, and if the flag has not been set, the user has never configured, and background can assume a reasonable default for the other flags.

**Comment by ChuckJHardy** [comment]
Working on a fix for the permissions. For the moment I have reverted all changes. Reloading the extension should solve this for now. Sorry for this.

**Comment by ChuckJHardy** [comment]
Thanks for everyones input - Next version will take all this into account.
